### PR TITLE
adidas ASCII logo: Go, Swift and Kotlin

### PR DESCRIPTION
**Pull request by josete89**

### CODE REVIEW

File: adidas-ascii-logo/go/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# adidas ASCII logo: TypeScript
+
+Implementation to resolve the [adidas ASCII logo challenge][adidas-ascii-logo] with Go.
+
+## Requirements and dependencies
+
+[Go][go] are required to work with the repository.
```
Comment (RecuencoJones) [inline adidas-ascii-logo/go/README.md]: `are` -> `is`

File: adidas-ascii-logo/go/main_test.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package main
+import (
+  "fmt"
+  "testing"
+)
+
+func TestWidth2(t *testing.T) {
```
Comment (RecuencoJones) [inline adidas-ascii-logo/go/main_test.go]: Can we add a couple more tests for go?

File: adidas-ascii-logo/swift/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# adidas ASCII logo: JavaScript
+
+Implementation to resolve the [adidas ASCII logo challenge][adidas-ascii-logo] with Swift.
+
+## Requirements and dependencies
+
+[Swift/Swift Package Manger][nodejs] are required to work with the repository.
```
Comment (RecuencoJones) [inline adidas-ascii-logo/swift/README.md]: I think we have to change `nodejs` to the link for `swift`'s homepage

Also, `are` -> `is`

File: adidas-ascii-logo/swift/Tests/adidas-ascii-logoTests/adidas_ascii_logoTests.swift [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+          @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+           @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+            @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@     @@@@@@@@@@@@@@@@@@@@@
+        """
+        XCTAssertEqual(createAdidasAsciiLogo(width:21),res)
+    }
+    
+
+    static var allTests = [
+        ("testWidth2", testWidth2),
```
Comment (RecuencoJones) [inline adidas-ascii-logo/swift/Tests/adidas-ascii-logoTests/adidas_ascii_logoTests.swift]: Should we add here all the other test cases?

File: adidas-ascii-logo/go/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (moelders) [inline adidas-ascii-logo/go/README.md]: Just one blank line.

File: adidas-ascii-logo/go/main.go [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+import (
+  "fmt"
+)
+func createAdidasAsciiLogo(width int) string {
+  return fmt.Sprintf("%d",width)
+}
+
+func main(){
+  fmt.Println(fmt.Sprintf("\nadidas (width %d)",2))
```
Comment (moelders) [inline adidas-ascii-logo/go/main.go]: If you shows only one example, use at least width `9`.

File: adidas-ascii-logo/go/main_test.go (hunk 2 above)
Comment (moelders) [reply adidas-ascii-logo/go/main_test.go]: You can find all test cases in JS and TS folders.

File: adidas-ascii-logo/swift/README.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
```
Comment (moelders) [inline adidas-ascii-logo/swift/README.md]: Just one blank line.

File: adidas-ascii-logo/swift/Sources/adidas-ascii-logo/adidas_ascii_logo.swift [hunk 8]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (moelders) [inline adidas-ascii-logo/swift/Sources/adidas-ascii-logo/adidas_ascii_logo.swift]: No blank line.

File: adidas-ascii-logo/swift/Tests/LinuxMain.swift [hunk 9]
```diff
@@ -0,0 +1,7 @@
+import XCTest
+
+import adidas_ascii_logoTests
+
+var tests = [XCTestCaseEntry]()
+tests += adidas_ascii_logoTests.allTests()
+XCTMain(tests)
```
Comment (moelders) [inline adidas-ascii-logo/swift/Tests/LinuxMain.swift]: It is recommended to add a blank line at the end of the file.

File: adidas-ascii-logo/swift/Tests/adidas-ascii-logoTests/XCTestManifests.swift [hunk 10]
```diff
@@ -0,0 +1,9 @@
+import XCTest
+
+#if !os(macOS)
+public func allTests() -> [XCTestCaseEntry] {
+    return [
+        testCase(adidas_ascii_logoTests.allTests),
+    ]
+}
+#endif
```
Comment (moelders) [inline adidas-ascii-logo/swift/Tests/adidas-ascii-logoTests/XCTestManifests.swift]: It is recommended to add a blank line at the end of the file.

File: adidas-ascii-logo/swift/Tests/LinuxMain.swift (hunk 9 above)
Comment (moelders) [reply adidas-ascii-logo/swift/Tests/LinuxMain.swift]: The `.editorconfig` is already set to do it.

File: adidas-ascii-logo/go/README.md (hunk 1 above)
Comment (josete89) [reply adidas-ascii-logo/go/README.md]: Done

File: adidas-ascii-logo/go/main_test.go (hunk 2 above)
Comment (josete89) [reply adidas-ascii-logo/go/main_test.go]: Done

File: adidas-ascii-logo/swift/README.md (hunk 3 above)
Comment (josete89) [reply adidas-ascii-logo/swift/README.md]: Done

File: adidas-ascii-logo/swift/Tests/adidas-ascii-logoTests/adidas_ascii_logoTests.swift (hunk 4 above)
Comment (josete89) [reply adidas-ascii-logo/swift/Tests/adidas-ascii-logoTests/adidas_ascii_logoTests.swift]: Done

File: adidas-ascii-logo/go/README.md (hunk 5 above)
Comment (josete89) [reply adidas-ascii-logo/go/README.md]: Done

File: adidas-ascii-logo/go/main.go (hunk 6 above)
Comment (josete89) [reply adidas-ascii-logo/go/main.go]: Done

File: adidas-ascii-logo/swift/README.md (hunk 7 above)
Comment (josete89) [reply adidas-ascii-logo/swift/README.md]: Done

File: adidas-ascii-logo/swift/Sources/adidas-ascii-logo/adidas_ascii_logo.swift (hunk 8 above)
Comment (josete89) [reply adidas-ascii-logo/swift/Sources/adidas-ascii-logo/adidas_ascii_logo.swift]: Done

File: adidas-ascii-logo/swift/Tests/LinuxMain.swift (hunk 9 above)
Comment (josete89) [reply adidas-ascii-logo/swift/Tests/LinuxMain.swift]: Done

File: adidas-ascii-logo/swift/Tests/adidas-ascii-logoTests/XCTestManifests.swift (hunk 10 above)
Comment (josete89) [reply adidas-ascii-logo/swift/Tests/adidas-ascii-logoTests/XCTestManifests.swift]: Done

**Pull request merged**